ui/v2.5/src/docs/en/Scraping.md

```diff
@@ -1,849 +1,56 @@
 # Metadata Scraping
 
-Stash supports scraping of performer, scene and movie details.
+Stash supports scraping of metadata from various external sources
 
-Stash includes a freeones.xxx performer scraper built in.
+## Terminology
 
-# Adding custom scrapers
+| Term | Description |
+|---|:---|:---|
```

Comment:
Extra column here is breaking the table

Reply:
fixed, thanks

 I'll wait to push it with any other fixes so as to avoid any unnecessary GH actions